redcachekeeper/keeper.go

```diff
@@ -23,6 +23,7 @@ type (
 		GetOrLock(string) (interface{}, *redsync.Mutex, error)
 		Store(*redsync.Mutex, Item) error
 		Purge(string) error
+		Increase(key string) error
```

Comment:
don, I need you to put more context on this.
Increase only used to increase cached integer. so I might need more context on the function. Be it the name, or documentation on when to user

Reply:
I think it would be both